lib/emojis.dart

```diff
@@ -1,25 +1,3 @@
-/// The skin tone used for emojis
-enum SkinTone {
-  ///  🤙🏻Light skin tone
-  light,
-
-  /// 🤙🏼 Medium Light skin tone
-  mediumLight,
-
-  /// 🤙🏽 Medium skin tone
-  medium,
-
-  /// 🤙🏾 Medium Dark skin tone
-  mediumDark,
-
-  /// 🤙🏿 Dark skin tone
-  dark,
-
-  /// 🤙 Default yellow skin tone.
-  @Deprecated('Use a value of null instead')
-  simpson,
-}
```

Comment:
I think the option to pass on skin tone color is much better as developer/user might store the preferred skin tone and we can just pass on this `enum`.

Else, we would have to map the preference with emoji skin color from `AnimatedEmojis.values` and it'll get too much complicated

Reply:
I agree but that would come up with one down side while importing json asset in pubspec.yaml

Like developer want to use muscle emoji with dark skin tone in asset json mode then he/she have to import file

> packages/animated_emoji/lottie/muscleDark.json

and have to use this in code

> AnimatedEmojis.muscle(SkinTone.dark)

but without this SkinTone enum

> AnimatedEmojis.muscleDark

that is same as asset file imported

Reply:
@hsbijarniya, sorry but I didn't understand. Can you please elaborate more?

Reply:
@OutdatedGuy I think he means that it would be more consistent when the name of the emoji in the assets always corresponds to the name used in flutter.

In addition, it would be confusing when importing for example `packages/animated_emoji/lottie/muscle.json` because this does not allow you to use all variations of `AnimatedEmojis.muscle` and the names in flutter do not make it clear what the other files should be called that you need to import to use all variations.